gotooltest/setup_test.go

```diff
@@ -0,0 +1,30 @@
+package gotooltest
+
+import (
+	"os"
+	"testing"
+)
+
+func TestInitGoEnv(t *testing.T) {
+	// Set up a temp directory containing a bad go.mod file to
+	// ensure the working directory does not influence the probe
+	// commands run during initGoEnv
```

Comment:
btw, here you should document that we assume the system's temp directory is not itself part of any Go module or workspace.

Reply:
Per next comment.